openshift/gabi.template.yaml

```diff
@@ -245,7 +249,9 @@ parameters:
 - name: OAUTH_PROXY_IMAGE_NAME
   value: quay.io/openshift/origin-oauth-proxy
 - name: OAUTH_PROXY_IMAGE_TAG
-  value: "4.10.0"
+  value: "4.14.0"
```

Comment:
**To the reviewer:**

Update to the latest stable release of the Red Hat's fork of OAuth2 Proxy.